.env.example

```
@@ -0,0 +1,11 @@
+# PostgreSQL
+  POSTGRES_USER=vow_pact
```

dotenv-linter (4.0.0)

[warning] [LeadingCharacter] Invalid leading character detected

(LeadingCharacter)